Source/Scene/BillboardCollection.js

```diff
@@ -119,7 +123,8 @@ define([
         pixelOffsetScaleByDistance : 8,
         compressedAttribute3 : 9,
         textureCoordinateBoundsOrLabelTranslate : 10,
-        a_batchId : 11
+        a_batchId : 11,
+        sdf: 12
```

Comment:
It might be.  As of right now I need 2 floats for the SDF information to store the outline color and the outline width.  I was doing my best to not step on too much that was already there, but if you have suggestions on where to stick it in another attribute that would be fine as well.

Reply:
@lilleyse any suggestion?

Reply:
I checked and all the attributes are full except the last component of `eyeOffset` when not instanced, two components in `textureCoordinateBoundsOrLabelTranslate` when VTF is supported, and three components in `a_batchId` when there's a batch table. It doesn't really seem ideal to put SDF in any of these due to all those conditions, so I think separate is good. Just add a comment next to the `sdf` property in `attributeLocationsBatched`/`attributeLocationsInstanced` that there are 8 bytes free.

Reply:
Actually maybe don't add the comment since the sdf attribute is only added when labels are using sdf, and we don't want to give the impression that any old attribute can use it.

Reply:
Thanks for looking into that.  I'll leave out the comment